flyway/scxa/migrations/V18__scxa-create-exp-design.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE exp_design
+(
+    accession VARCHAR(255)
+        CONSTRAINT exp_design_k
+            REFERENCES experiment (accession)
+            ON DELETE CASCADE,
+    marray VARCHAR(255),
+    sample VARCHAR(255),
+    annot_type char(50)
+        CONSTRAINT exp_design_annot_type_check
+            CHECK (annot_type = ANY (ARRAY['characteristic'::bpchar, 'factor'::bpchar])),
+    annot VARCHAR(255),
+    annot_value VARCHAR(255),
+    annot_ont_URI VARCHAR(255),
+    CONSTRAINT exp_design_pk
+        UNIQUE (accession, sample, annot_type)
+);
+
+CREATE TABLE exp_design_column_sequence
```

Comment:
Just call this 'exp_design_column' (assuming others approve my plan above), reflecting its more generic puprpose.